### PR TITLE
ZK-4920: listbox (autopaging) listitem height incorrect between icebl…

### PR DESCRIPTION
**Pull request by plijyfes**
…ue/iceblue_c

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -3010,6 +3010,7 @@ B90-ZK-4431.zul=A,E,Multislider
 ##zats##B96-ZK-4728.zul=A,E,Tree,Treeitem,paging,visible,open
 ##ztl##B96-ZK-4929.zul=A,E,Bandbox,Keyboard,Browser
 ##manually##B96-ZK-4935.zul=A,E,JakartaEE,FileUpload,Content-Length
+##manually##B96-ZK-4920.zul=A,E,MeshWidget,autopaging,paging,CSS,Style,Themes
```
Comment (plijyfes) [inline zktest/src/archive/test2/config.properties:3013]: add ZATS

**Review by scribetw — CHANGES_REQUESTED** [review]
refine needed

**Review by scribetw — APPROVED** [review]

**Review by DevChu — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes and merged, enjoy!

**Pull request merged**